x-pack/plugins/cases/public/components/header_page/title.tsx

```diff
@@ -30,7 +32,7 @@ interface Props {
 const TitleComponent: React.FC<Props> = ({ title, badgeOptions }) => (
   <EuiTitle size="l">
     <h1 data-test-subj="header-page-title">
-      {title}
+      {isString(title) ? <TruncatedText text={title} /> : { title }}
```

Comment:
I believe you just need `title`, not `{ title }` since you are already inside the `{}`. This will  attempt to render as an object and throw an error in the browser. 
```suggestion
      {isString(title) ? <TruncatedText text={title} /> : title }
```

Reply:
You are right. Fixed!